specs/WinRT/WinRTAPIContracts.md

```diff
@@ -62,27 +74,49 @@ namespace Microsoft.Windows.System
+4. Contracts/Types in releases honor the source/binary/behavior compatibility rules e.g.
 contracts/types in a stable release doesn't change in future releases of the same major version
 (e.g. contracts/types in 1.1 don't change in 1.2, 1.3, ...)
 
 The builds that check for contracts compatibility are:
 1. Nightly
-2. Release 
+2. Release
 3. PR
 
-## Open issues
-None.
+# 5. Telemetry
+API contract version specific telemetry will not be collected because it does not provide any
+additional information compared to the Windows App SDK version.
+
+# 6. All Contracts in Windows App SDK
+The list of all contracts defined across Windows App SDK
+
+| Feature            | Repository    | Contract                        | Namespace                                            | Comment |
+|--------------------|---------------|---------------------------------|------------------------------------------------------|---------|
+| AccessControl      | windowsappsdk | AccessControlContract           | Microsoft.Windows.Security.AccessControl             |         |
+| AppLifecycle       | windowsappsdk | AppLifecycleContract            | Microsoft.Windows.AppLifecycle                       |         |
+| AppNotifications   | windowsappsdk | AppNotificationsContract        | Microsoft.Windows.AppNotifications                   |         |
+| Deployment         | windowsappsdk | DeploymentManagerContract       | Microsoft.Windows.ApplicationModel.WindowsAppRuntime |         |
+| DynamicDependency  | windowsappsdk | DynamicDependencyContract       | Microsoft.Windows.ApplicationModel.DynamicDependency |         |
+| EnvironmentManager | windowsappsdk | EnvironmentManagerContracct     | Microsoft.Windows.System                             |         |
+| MRTCore            | windowsappsdk | MrtContract                     | Microsoft.Windows.ApplicationModel.Resources         |         |
+| PowerNotifications | windowsappsdk | PowerNotificationsContract      | Microsoft.Windows.System.Power                       |         |
+| PushNotifications  | windowsappsdk | PushNotificationsContract       | Microsoft.Windows.PushNotifications                  |         |
+| WinUI              | winui         | HostingContract                 | Microsoft.UI.Xaml.Hosting                            |         |
+| WinUI              | winui         | WinUIContract                   | Microsoft.UI.Xaml                                    |         |
+| WinUI              | winui         | WinUIControlsContract           | Microsoft.UI.Xaml.Controls                           |         |
+| WinUI              | winui         | XamlDirectContract              | Microsoft.UI.Xaml.Core.Direct                        |         |
+| ---                | ---           | WindowsAppSDKContract           | Microsoft.Foundation                                 |1,2,3    |
+
+<sup>1</sup> Only for use by IXP.<BR>
```

Comment:
It's not a secret. "ixp" appears in multiple places

WinAppSDK's nuget `build\Microsoft.InteractiveExperiences.targets`

```xml
     <CsWinRTInputs Include="$(MSBuildThisFileDirectory)..\lib\$(Ixp-UAPTargetVersion)\*.winmd" />
```

Our nuspec
```xml
    <tags>...,Interactive Experiences,ixp,...</tags>
```

But here it's only referenced once in the footnote. That can easily be changed to 'InteractiveExperiences' (and should be, since the abbreviation's not defined anywhere, no matter how commonly informally used)